wordpress-mobile/WordPress-iOS
# Update Helpshift to version 6.3.0

**Fixes** #7979 

**To test:**
 - Start the app
 - Go to Me -> Help & Support -> Contact Us
 - Check that chat is working ok.
 - Check that ticket deflection is working ok.
 - Test that all tests pass (the one disabled here: https://github.com/wordpress-mobile/WordPress-iOS/pull/7897 is now enabled).

Needs review: @SergioEstevao
